hip/components/atomic.hip.hpp

```diff
@@ -62,13 +63,14 @@ __forceinline__ __device__ void atomic_add(
  *
  * @note It is not 'real' complex<double> atomic add operation
  */
-__forceinline__ __device__ void atomic_add(
+__forceinline__ __device__ thrust::complex<double> atomic_add(
     thrust::complex<double> *__restrict__ address, thrust::complex<double> val)
 {
     hipDoubleComplex *cuaddr = reinterpret_cast<hipDoubleComplex *>(address);
```

Comment:
`cuaddr` -> `hipaddr` (is not changed by script)
I do not write this rule in `cuda2hip` because it is not general

Or, `cuaddr` or `hipaddr` -> `addr` is also fine

Reply:
done